### PR TITLE
cli: replace program.rpc with program.methods in mocha test template

### LINKED ISSUE #1463: cli: mocha test template should use program.methods not program.rpc

### PR DESCRIPTION
**Pull request by NBNARADHYA**
Signed-off-by: Nikhil B N <nikhilbn365@gmail.com>

Fixes #1463 

`program.rpc` will be deprecated eventually

### CODE REVIEW

File: cli/src/template.rs [hunk 1]
```diff
@@ -201,7 +201,7 @@ describe("{}", () => {{
   it("Is initialized!", async () => {{
     // Add your test here.
     const program = anchor.workspace.{};
-    const tx = await program.rpc.initialize();
+    const tx = await program.methods.initialize();
```
Comment (armaniferrante) [inline cli/src/template.rs:204]: Need to change to `program.methods.initialize().rpc()`

Comment (NBNARADHYA) [reply cli/src/template.rs]: Done!

**Comment by paul-schaaf** [comment]
hey @NBNARADHYA do you have time to finish this or would you like me to take over?

**Comment by paul-schaaf** [comment]
closed by #1732

**Comment by NBNARADHYA** [comment]
> hey @NBNARADHYA do you have time to finish this or would you like me to take over?

@paul-schaaf I apologise for not replying --> Was having exams.

**Pull request closed without merging**